src/styles/block/stars.scss

```diff
@@ -0,0 +1,36 @@
+.stars {
+  justify-content: space-between;
+  display: flex;
```

Comment:
Properties within a selector should be ordered consistently. According to the code style guide, 'display' should come before 'justify-content'.